solutions/landing-zone/README.md

```diff
@@ -88,13 +88,19 @@ To deploy this Landing Zone you will first need to create a Bootstrap project wi
 
 0. Set Default Logging Storage Location.
 
-    This command will ensure that the default logging buckets that are generated with a new project are set to the selected region instead of the default location `global`.
+    This command will ensure that the default logging buckets that are generated with a new project (organization wide) are set to the selected region instead of the default location `global`.
 
     ```
+    export REGION=northamerica-northeast1
+    export PROJECT_ID=$(gcloud config list --format 'value(core.project)')
+    export ORG_ID=$(gcloud projects get-ancestors $PROJECT_ID --format='get(id)' | tail -1)
+    export EMAIL=your-super-admin@email.com
+    gcloud organizations add-iam-policy-binding "${ORG_ID}" --member "user:${EMAIL}" --role roles/logging.admin
+    
     cloud alpha logging settings update --organization=$ORG_ID --storage-location=$REGION
```

Comment:
might as well change "cloud" to "gcloud"

Reply:
Dave, that is perfect - very good eye, modifying